geth/jail/execution_policy.go

```diff
@@ -30,18 +32,58 @@ func NewExecutionPolicy(nodeManager common.NodeManager, accountManager common.Ac
 	}
 }
 
-// ExecuteSendTransaction defines a function to execute RPC requests for eth_sendTransaction method only.
-func (ep *ExecutionPolicy) ExecuteSendTransaction(req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
+// Execute handles the execution of a RPC request
+func (ep *ExecutionPolicy) Execute(req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
 	config, err := ep.nodeManager.NodeConfig()
 	if err != nil {
 		return nil, err
 	}
 
-	if config.UpstreamConfig.Enabled {
-		return ep.executeRemoteSendTransaction(req, call)
+	var res *otto.Object
+	var resErr error
+
+	switch {
+	case strings.HasPrefix(req.Method, "shh_"):
+		res, resErr = ep.ExecuteLocally(req, call)
+	case params.SendTransactionMethodName == req.Method:
+		switch config.UpstreamConfig.Enabled {
+		case true:
+			res, resErr = ep.executeRemoteSendTransaction(req, call)
```

Comment:
Any reason why `execute*SendTransaction` methods are not exported while `ExecuteLocally` and `ExecuteOnRemote` are exported?

Reply:
`execute*sendTransaction` are unexported, has I felt a single `ExecuteSendTransaction` would be a better encapsulation for the different needed implementation for upstream/local.

Reply:
`execute*sendTransaction` are unexported, has I felt a single `ExecuteSendTransaction` would be a better encapsulation for the different needed implementation for upstream/local.